reference/admin.yaml

```diff
@@ -12836,6 +12836,11 @@ paths:
               properties:
                 data:
                   $ref: '#/components/schemas/report_action_with_expenses_ids_to_eject'
+                reason:
+                  type: string
+                  description: |
+                    If admin tries to remove expenses from other's report then reason is required, otherwise reason is not required.
```

Comment:
Since this is admin API, can we rephrase the sentence to something like: 
```
Reason for ejecting the expense
Note: When ejecting own expense, reason is not required. However, if another spender's expense are ejected reason is mandatory
```